src/wwwroot/css/styles.css

```diff
@@ -152,6 +152,7 @@ body {
 }
 
 .table-wrap {
-    height: 80px;
+    height: 50%;
     overflow-y: auto;
 }
+
```

Comment:
Can' comment, but on line 31 in this file Visual Studio is telling there is an error, though it still compiles and runs fine

Reply:
Since it is _CSS_ it will always compile, but in case of error it wont display something right, I'll take a look and try to fix it, thanks!